umm-spec-lib/src/cmr/umm_spec/migration/version/core.clj

```diff
@@ -0,0 +1,67 @@
+(ns cmr.umm-spec.migration.version.core
+  "Contains functions for migrating between versions of UMM schema."
+  (:require
+   [clojure.set :as set]
+   [clojure.string :as string]
+   [cmr.common-app.services.kms-fetcher :as kf]
+   [cmr.common.mime-types :as mt]
+   [cmr.common.util :as util :refer [update-in-each]]
+   [cmr.umm-spec.dif-util :as dif-util]
+   [cmr.umm-spec.location-keywords :as lk]
+   [cmr.umm-spec.migration.collection-progress-migration :as coll-progress-migration]
+   [cmr.umm-spec.migration.contact-information-migration :as ci]
+   [cmr.umm-spec.migration.distance-units-migration :as distance-units-migration]
+   [cmr.umm-spec.migration.geographic-coordinate-units-migration :as geographic-coordinate-units-migration]
+   [cmr.umm-spec.migration.organization-personnel-migration :as op]
+   [cmr.umm-spec.migration.related-url-migration :as related-url]
+   [cmr.umm-spec.migration.spatial-extent-migration :as spatial-extent]
+   [cmr.umm-spec.migration.version.interface :as interface]
+   [cmr.umm-spec.spatial-conversion :as spatial-conversion]
+   [cmr.umm-spec.util :as u]
+   [cmr.umm-spec.versioning :refer [versions current-version]]
+   [cmr.umm-spec.xml-to-umm-mappings.characteristics-data-type-normalization :as char-data-type-normalization]))
```

Comment:
Should require namespaces `cmr.umm-spec.migration.version.collection` and `cmr.umm-spec.migration.version.variable` to support multimethod `interface/migrate-umm-version`. This is why the ci tests are failing.

Reply:
Nice catch.

Reply:
Fixed, pushed, and waiting for build ...